### DISCUSSION TITLE
404 problems

### DISCUSSION THREAD

**Discussion by 99zardoz** [community/community#141986] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi

I created a github pages site using the Jekyll modernist theme and it works OK locally when tested using Jekyll.  It has index.md in the root and is deployed from the root of the main branch.  Yet when github deploys it, I get  a 404.  I tried adding a nojekyll file and it makes no difference.   I have read the docs but not got anywhere.  Any help appreciated and my apologies if I overlooked something simple.

Site url: https://99zardoz.github.io/github.io/

Repository contents:

![image](https://github.com/user-attachments/assets/a536f37d-828f-4221-b26c-b916018e08bc)

Pages section

![image](https://github.com/user-attachments/assets/fd074206-b43b-4def-af64-9ba93aa95968)

**Comment by davevad93** [comment, marked as answer]
Hi @99zardoz ,

By default GitHub Pages will look for an `index.html`, `index.md`, or `README.md` file as the entry file for your site, as I can see from your screenshot, you have a file named `Index.md`, rename it to `index.md`; that's important because GitHub Pages is case sensitive so you should always name your index files in lower case.

**Reply by 99zardoz** [reply]
Thank you very much for your help.  In my local folder, it was index.md lower case i.  I had to rename it in github directly and that worked.  Much appreciate you taking the time to help.

**Reply by davevad93** [reply]
You're welcome @99zardoz , have a nice day! 👍
